lib/node.js

```diff
@@ -271,14 +269,19 @@ Node.prototype._syncBitcoind = function() {
       }
     });
   }, function(err) {
-    log.info('Stopping Bitcoind Sync');
+
     self.bitcoindSyncing = false;
     self.chain.lastSavedMetadataThreshold = 0;
     if (err) {
       Error.captureStackTrace(err);
       return self.emit('error', err);
     }
-    self.emit('synced');
+
+    // If bitcoind is completely synced
+    if (self.bitcoind.isSynced()) {
+      self.emit('synced');
+    }
+
```

Comment:
Ok workaround reached: we will keep emitting "synced" events as new blocks come in, but if you are interested in the initial block download, you would do:

```
node.once('synced', function() {
  // do stuff here
});
```
If we run into issues with this approach later, we can change it, but this is fine for now.